test/functional/http/get/peers.js

```diff
@@ -1,494 +1,157 @@
+				.then(function (res) {
+					res.body.peers[0].broadhash.should.be.eql(validHeaders.broadhash);
 				});
-			});
 		});
 
-		describe('limit', function () {
-
-			it('using string limit should fail', function (done) {
-				var limit = 'one';
-				var params = 'limit=' + limit;
-
-				http.get('/api/peers?' + params, function (err, res) {
-					node.expect(res.body).to.have.property('message').to.equal('Expected type integer but found type string');
-					done();
-				});
-			});
-
-			it('using limit = -1 should fail', function (done) {
-				var limit = -1;
-				var params = 'limit=' + limit;
-
-				http.get('/api/peers?' + params, function (err, res) {
-					node.expect(res.body).to.have.property('message').to.equal('Value -1 is less than minimum 1');
-					done();
-				});
-			});
-
-			it('using limit = 0 should fail', function (done) {
-				var limit = 0;
-				var params = 'limit=' + limit;
-
-				http.get('/api/peers?' + params, function (err, res) {
-					node.expect(res.body).to.have.property('message').to.equal('Value 0 is less than minimum 1');
-					done();
-				});
-			});
-
-			it('using limit = 1 should be ok', function (done) {
-				var limit = 1;
-				var params = 'limit=' + limit;
-
-				http.get('/api/peers?' + params, function (err, res) {
-					node.expect(res.body).to.have.property('peers').that.is.an('array');
-					node.expect(res.body.peers.length).to.be.at.most(limit);
-					done();
+		it('using sort = "version:asc" should be ok', function () {
+			return peersEndpoint.makeRequest({sort: 'version:asc'}, 200)
+				.then(function (res) {
+					_.orderBy(_.clone(res.body.peers), ['version'], ['asc']).should.be.eql(res.body.peers);
 				});
-			});
-
-			it('using limit = 100 should be ok', function (done) {
-				var limit = 100;
-				var params = 'limit=' + limit;
-
-				http.get('/api/peers?' + params, function (err, res) {
-					node.expect(res.body).to.have.property('peers').that.is.an('array');
-					node.expect(res.body.peers.length).to.be.at.most(limit);
-					done();
-				});
-			});
-
-			it('using limit > 100 should fail', function (done) {
-				var limit = 101;
-				var params = 'limit=' + limit;
-
-				http.get('/api/peers?' + params, function (err, res) {
-					node.expect(res.body).to.have.property('message').to.equal('Value 101 is greater than maximum 100');
-					done();
-				});
-			});
 		});
 
-		describe('offset', function () {
-
-			it('using string offset should fail', function (done) {
-				var offset = 'one';
-				var params = 'offset=' + offset;
-
-				http.get('/api/peers?' + params, function (err, res) {
-					node.expect(res.body).to.have.property('message').to.equal('Expected type integer but found type string');
-					done();
+		it('using sort = "version:desc" should be ok', function () {
+			return peersEndpoint.makeRequest({sort: 'version:desc'}, 200)
+				.then(function (res) {
+					_.orderBy(_.clone(res.body.peers), ['version'], ['desc']).should.be.eql(res.body.peers);
```

Comment:
In the rest of files to test a proper sort order we are using construction with: `_(res.body.peers).map('version').sort().value()` with optional `reverse()`. 

Please apply it to the file.

Reply:
Done.